pyproject.toml

```diff
@@ -9,6 +9,7 @@ dependencies = [
     "faiss-cpu~=1.11.0",
     "fire~=0.7.0",
     "lightspeed-stack-providers~=0.1.9",
```

Comment:
When [this](https://github.com/lightspeed-core/lightspeed-providers/pull/19) is merged I'll release a new version of `lightspeed-stack-providers` version `0.1.10` and this can be updated.